docs/userGuide/components/dropdown.md

```diff
@@ -0,0 +1,113 @@
+      </button>
+      <ul slot="dropdown-menu" class="dropdown-menu">
+        <li><a href="#dropdown">Action</a></li>
+        <li><a href="#dropdown">Another action</a></li>
+        <li><a href="#dropdown">Something else here</a></li>
+        <li role="separator" class="divider"></li>
+        <li><a href="#dropdown">Separated link</a></li>
+      </ul>
+    </dropdown>
+    <dropdown text="Action" type="primary">
+      <li><a href="#dropdown">Action</a></li>
+      <li><a href="#dropdown">Another action</a></li>
+      <li><a href="#dropdown">Something else here</a></li>
+      <li role="separator" class="divider"></li>
+      <li><a href="#dropdown">Separated link</a></li>
+    </dropdown>
+    <dropdown>
+      <button slot="button" type="button" class="btn btn-success dropdown-toggle">
+        Action <span class="caret"></span>
+      </button>
+      <ul slot="dropdown-menu" class="dropdown-menu">
+        <li><a href="#dropdown">Action</a></li>
+        <li><a href="#dropdown">Another action</a></li>
+        <li><a href="#dropdown">Something else here</a></li>
+        <li role="separator" class="divider"></li>
+        <li><a href="#dropdown">Separated link</a></li>
+      </ul>
+    </dropdown>
+    <dropdown text="Disabled" type="warning" disabled>
+      <li><a href="#dropdown">Action</a></li>
+    </dropdown>
+    <dropdown type="info">
+      <button slot="before" type="button" class="btn btn-info">Segmented</button>
+      <li><a href="#dropdown">Action</a></li>
+    </dropdown>
+    <div><br></div>
+    <div class="btn-group btn-group-justified">
+      <a href="#dropdown" class="btn btn-default" role="button">Left</a>
+      <dropdown>
+        <a slot="button" href="#dropdown" class="btn btn-default">
+          Dropdown <span class="caret"></span>
+        </a>
+        <ul slot="dropdown-menu" class="dropdown-menu">
+          <li><a href="#dropdown">Action</a></li>
+          <li><a href="#dropdown">Another action</a></li>
+          <li><a href="#dropdown">Something else here</a></li>
+          <li role="separator" class="divider"></li>
+          <li><a href="#dropdown">Separated link</a></li>
+        </ul>
+      </dropdown>
+      <a href="#dropdown" class="btn btn-default" role="button">Right</a>
+    </div>
+</tip-box>
+
+<tip-box border-left-color="black">
+<i style="font-style: normal; font-weight: bold; color: dimgray">Markup</i>
+
+```html
+<dropdown text="Action" type="primary">
+  <li><a href="#dropdown">Action</a></li>
+  <li><a href="#dropdown">Another action</a></li>
+  <li><a href="#dropdown">Something else here</a></li>
+  <li role="separator" class="divider"></li>
+  <li><a href="#dropdown">Separated link</a></li>
+</dropdown>
+<!-- For segmented dropdown, ignore text and add a "before" slot -->
+<dropdown type="info">
+  <button slot="before" type="button" class="btn btn-info">Segmented</button>
+  <li><a href="#dropdown">...</a></li>
+</dropdown>
+<div><br></div>

```

Comment:
Putting this in the Markup is not necessary, it just helps to make the example look more presentable.